plugins/modules/dcnm_network.py

```diff
@@ -181,11 +181,19 @@
           deploy:
             description:
             - Per switch knob to control whether to deploy the attachment
+            - This knob has been deprecated from release 2.1.0 onwards and will not have any functional
+              impact if specified in playbook.
             type: bool
             default: true
       deploy:
         description:
         - Global knob to control whether to deploy the attachment
+        - VERSION 2.0.1 AND BEFORE
```

Comment:
Please specify this as `Ansible NDFC Collection Behavior for Version 2.0.1 and earlier`

Reply:
Changes done as suggested